crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -153,7 +153,33 @@ impl Analyzer<'_, '_> {
 
             {
                 match ty.normalize() {
-                    Type::Ref(_) => {
+                    Type::Ref(ref_ty) => {
+                        if let Some(ty_args) = &ref_ty.type_args {
+                            if !ty_args.params.is_empty() {
+                                if let RTsEntityName::Ident(id) = &ref_ty.type_name {
+                                    if let Ok(Some(ty)) = &self.find_type(&id.into()) {
+                                        let ty_found = &ty.clone().map(|v| v.into_owned()).collect::<Vec<Type>>()[0];
+
+                                        match ty_found.normalize() {
+                                            Type::Class(..) | Type::ClassDef(..) | Type::Interface(..) | Type::Alias(..) => {
```

Comment:
changed to match https://github.com/dudykr/stc/blob/5ec6c775d520bd2eef9f6f7ee0ea3f83eeccd484/crates/stc_ts_types/src/lib.rs#L2359